### PR TITLE
Upgrade Profile Header to Vanilla v1

### PR DESCRIPTION
**Pull request by barrymcgee**

### CODE REVIEW

**Comment by barrymcgee** [comment]
## QA

Before this change...

screen shot 2018-08-10 at 12 44 48

After this change...

screen shot 2018-08-10 at 12 46 12

**Comment by anthonydillon** [comment]
Is this 1.8.0? Take headings should be uppercase and grey.

File: jujugui/static/gui/src/app/components/profile/header/_header.scss [hunk 1]
```diff
@@ -1,126 +1,13 @@
 .profile-header {
-  background-color: $white;
-  padding-top: 30px;
-  padding-bottom: 30px;
-  border-bottom: 1px solid $light-grey;
+  margin-bottom: 3rem;
 
-  &__inner {
-    display: flex;
+  .p-strip {
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/profile/header/_header.scss]: Isn't `p-sprip--light` white.

File: jujugui/static/gui/src/app/components/profile/header/_header.scss [hunk 2]
```diff
@@ -1,126 +1,13 @@
 .profile-header {
-  background-color: $white;
-  padding-top: 30px;
-  padding-bottom: 30px;
-  border-bottom: 1px solid $light-grey;
+  margin-bottom: 3rem;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/profile/header/_header.scss]: Can you comment that this is to compensate for the lack of a following `p-strip`

File: jujugui/static/gui/src/app/components/profile/header/header.js [hunk 3]
```diff
@@ -104,52 +103,66 @@ class ProfileHeader extends React.Component {
...
-              name="close_16"
-              size="20" />
+      <div className="profile-header v1">
+        <div className="p-strip is-shallow">
+          <div className="row p-divider u-no-padding--top u-no-padding--bottom">
+            <div className="col-10 p-divider__block">
+              <div className="p-media-object">
+                {this._generateAvatar()}
+                <div className="p-media-object__details">
+                  <h1>
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/profile/header/header.js]: Shouldn't this have the media object styling? p-media-object__title

File: jujugui/static/gui/src/app/components/profile/header/header.js [hunk 4]
```diff
@@ -104,52 +103,66 @@ class ProfileHeader extends React.Component {
...
-            role="button"
-            tabIndex="0">
-            <SvgIcon
-              name="close_16"
-              size="20" />
+      <div className="profile-header v1">
+        <div className="p-strip is-shallow">
+          <div className="row p-divider u-no-padding--top u-no-padding--bottom">
+            <div className="col-10 p-divider__block">
+              <div className="p-media-object">
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/profile/header/header.js]: Shouldn't this be the large version?

File: jujugui/static/gui/src/app/components/profile/header/_header.scss (hunk 1 above)
Comment (barrymcgee) [reply jujugui/static/gui/src/app/components/profile/header/_header.scss]: No, it's grey - https://docs.vanillaframework.io/en/patterns/strip

**Comment by barrymcgee** [comment]
@anthonydillon Ready for re-review

**Comment by barrymcgee** [comment]
`vanilla-framework@1.8.0` is now added as an NPM package. Legacy `cloud-vanilla-theme` has been added manually as a plugin and as it relies on `vanilla framework@0.1.0`, that has also been added to sit along side it. No more changes will be made to either of these repos, they are there purely to support components which have not yet been converted. When the conversion is complete, we'll just remove them entirely.

**Comment by spencerbygraves** [comment]
@barrymcgee I can't get this to work, can you add a screen grab or two please. Thanks.

**Comment by hatched** [comment]
@huwshimi could you plz take a look at this

**Comment by huwshimi** [comment]
This may be intentional, but there is now more space at the top and bottom of the header (even from your screenshot above):

header

**Comment by barrymcgee** [comment]
@huwshimi Yes, that's intentional as the extra space is needed to accommodate the tooltip to edit your Gravatar when hovering the avatar.

**Comment by spencerbygraves** [comment]
@barrymcgee Is there a way we can avoid adding the extra vertical space?

**Comment by barrymcgee** [comment]
@spencerbygraves Not really, head space is needed for the Gravarat tooltip

screen shot 2018-08-16 at 10 39 18

**Comment by spencerbygraves** [comment]
@barrymcgee would it help to have it at the bottom?

image

**Comment by barrymcgee** [comment]
screen shot 2018-08-21 at 09 30 14

**Comment by lyubomir-popov** [comment]
can't add a plus one for some reason, but Barry showed it to me via screenshare. 

LGTM.

**Comment by hatched** [comment]
Thanks @barrymcgee

**Pull request merged**